x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/exceptions/exceptions_list_card.tsx

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo, useState } from 'react';
+import styled from 'styled-components';
+
+import {
+  EuiBadge,
+  EuiLink,
+  EuiButtonIcon,
+  EuiContextMenuItem,
+  EuiContextMenuPanel,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiTextColor,
+  EuiPanel,
+  EuiPopover,
+  EuiText,
+} from '@elastic/eui';
+import type { HttpSetup } from '@kbn/core-http-browser';
+import type { NamespaceType } from '@kbn/securitysolution-io-ts-list-types';
+import type { ExceptionListInfo } from './use_all_exception_lists';
+
+const StyledFlexItem = styled(EuiFlexItem)`
+  border-right: 1px solid #d3dae6;
+  padding: 4px 12px 4px 0;
+`;
+
+interface ExceptionsListCardProps {
+  exceptionsList: ExceptionListInfo;
+  http: HttpSetup;
+  handleDelete: ({
+    id,
+    listId,
+    namespaceType,
+  }: {
+    id: string;
+    listId: string;
+    namespaceType: NamespaceType;
+  }) => () => Promise<void>;
+  handleExport: ({
+    id,
+    listId,
+    namespaceType,
+  }: {
+    id: string;
+    listId: string;
+    namespaceType: NamespaceType;
+  }) => () => Promise<void>;
+  readOnly: boolean;
+}
+
+export const ExceptionsListCard = memo<ExceptionsListCardProps>(
+  ({ exceptionsList, http, handleDelete, handleExport, readOnly }) => {
+    const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+
+    const onItemActionsClick = () => setIsPopoverOpen((isOpen) => !isOpen);
+    const onClosePopover = () => setIsPopoverOpen(false);
+
+    return (
+      <EuiFlexGroup>
```

Comment:
```suggestion
      <EuiFlexGroup>
        <EuiFlexItem>
          <EuiPanel>
            <EuiFlexGroup key={exceptionsList.list_id} alignItems="center" gutterSize="l">
              <EuiFlexItem grow={true}>
                <EuiFlexGroup gutterSize="none" direction="column">
                  <EuiFlexItem grow={true}>
                    <EuiTitle size="xs">
                      <EuiLink data-test-subj="exception-list-name">
                        {exceptionsList.name.toString()}
                      </EuiLink>
                    </EuiTitle>
                  </EuiFlexItem>
                  <EuiFlexItem grow={false}>
                    <EuiText size="s">
                      <EuiTextColor color="subdued">{exceptionsList.description}</EuiTextColor>
                    </EuiText>
                  </EuiFlexItem>
                </EuiFlexGroup>
              </EuiFlexItem>

              <EuiFlexItem grow={false}>
                <EuiFlexGroup gutterSize="s" alignItems="center">
                  <StyledFlexItem style={{ flex: '1 1 auto' }}>
                    <EuiFlexGroup gutterSize="xs" alignItems="center">
                      <EuiFlexItem grow={false}>
                        <EuiTitle size="xxs">
                          <h6>{i18n.DATE_CREATED_LABEL}</h6>
                        </EuiTitle>
                      </EuiFlexItem>
                      <EuiFlexItem grow={false}>
                        <EuiBadge>{exceptionsList.created_at}</EuiBadge>
                      </EuiFlexItem>
                    </EuiFlexGroup>
                  </StyledFlexItem>
                  <StyledFlexItem style={{ flex: '1 1 auto' }}>
                    <EuiFlexGroup gutterSize="xs" alignItems="center">
                      <EuiFlexItem grow={false}>
                        <EuiTitle size="xxs">
                          <h6>{i18n.CREATED_BY_LABEL}</h6>
                        </EuiTitle>
                      </EuiFlexItem>
                      <EuiFlexItem grow={false}>
                        <EuiBadge>{exceptionsList.created_by}</EuiBadge>
                      </EuiFlexItem>
                    </EuiFlexGroup>
                  </StyledFlexItem>
                  <StyledFlexItem style={{ flex: '1 1 auto' }}>
                    <EuiFlexGroup gutterSize="xs" alignItems="center">
                      <EuiFlexItem grow={false}>
                        <EuiTitle size="xxs">
                          <h6>{i18n.EXCEPTIONS_LABEL}</h6>
                        </EuiTitle>
                      </EuiFlexItem>
                      <EuiFlexItem grow={false}>
                        <EuiBadge>{'3'}</EuiBadge>
                      </EuiFlexItem>
                    </EuiFlexGroup>
                  </StyledFlexItem>
                  <EuiFlexItem style={{ flex: '1 1 auto' }}>
                    <EuiFlexGroup gutterSize="xs" alignItems="center">
                      <EuiFlexItem grow={false}>
                        <EuiTitle size="xxs">
                          <h6>{i18n.RULES_LABEL}</h6>
                        </EuiTitle>
                      </EuiFlexItem>
                      <EuiFlexItem grow={false}>
                        <EuiBadge>{'3'}</EuiBadge>
                      </EuiFlexItem>
                    </EuiFlexGroup>
                  </EuiFlexItem>
                  <EuiFlexItem grow={false}>
                    <EuiPopover
                      data-test-subj="exceptionsListCardOverflowActions"
                      button={
                        <EuiButtonIcon
                          isDisabled={false}
                          data-test-subj="exceptionsListCardOverflowActions"
                          aria-label="Exception item actions menu"
                          iconType="boxesHorizontal"
                          onClick={onItemActionsClick}
                        />
                      }
                      panelPaddingSize="none"
                      isOpen={isPopoverOpen}
                      closePopover={onClosePopover}
                    >
                      <EuiContextMenuPanel
                        size="s"
                        items={[
                          <EuiContextMenuItem
                            key={'delete'}
                            disabled={exceptionsList.list_id === 'endpoint_list' || readOnly}
                            data-test-subj="exceptionsTableDeleteButton"
                            icon={'trash'}
                            onClick={() => {
                              onClosePopover();
                              handleDelete({
                                id: exceptionsList.id,
                                listId: exceptionsList.list_id,
                                namespaceType: exceptionsList.namespace_type,
                              })();
                            }}
                          >
                            {'Delete exception list'}
                          </EuiContextMenuItem>,
                          <EuiContextMenuItem
                            key={'export'}
                            icon={'exportAction'}
                            data-test-subj="exceptionsTableExportButton"
                            onClick={() => {
                              onClosePopover();
                              handleExport({
                                id: exceptionsList.id,
                                listId: exceptionsList.list_id,
                                namespaceType: exceptionsList.namespace_type,
                              })();
                            }}
                          >
                            {'Export exception list'}
                          </EuiContextMenuItem>,
                        ]}
                      />
                    </EuiPopover>
                  </EuiFlexItem>
                </EuiFlexGroup>
              </EuiFlexItem>
            </EuiFlexGroup>
          </EuiPanel>
        </EuiFlexItem>
      </EuiFlexGroup>
```

We'd chatted about the spacing being off, this should fix that and get it a bit closer to the designs. The hardcoded parts in my suggestion need updates though (the number of exception items and associated rules count).

<img width="1487" alt="Screen Shot 2022-10-07 at 12 34 40 PM" src="https://user-images.githubusercontent.com/10927944/194640264-0226120a-3ea7-49b0-8075-bbea141197e4.png">

Reply:
Worked with @XavierM before you commented on these but we came up with the following updates to fix most of the spacing. Not as compact as yours though. Wondering how much compactness would be preferable here.


<img width="1160" alt="Screen Shot 2022-10-25 at 3 43 45 PM" src="https://user-images.githubusercontent.com/915763/197867283-2a218bcc-1fe2-47db-8585-c4156202df20.png">